dissect/target/loaders/velociraptor.py

```diff
@@ -14,17 +14,21 @@
 
 def find_fs_directories(path: Path) -> tuple[Optional[OperatingSystem], Optional[list[Path]]]:
     # As of Velociraptor version 0.6.7 the structure of the Velociraptor Offline Collector varies by operating system
-    # Generic.Collectors.File (Unix, OS-X) root filesystem is 'uploads/'
+    # Generic.Collectors.File (Linux, Unix and OS-X) root filesystem is 'uploads/file/'
     # Generic.Collectors.File (Windows) and Windows.KapeFiles.Targets (Windows) root filesystem is
     # 'uploads/<file-accessor>/<drive-name>/'
     fs_root = path.joinpath(FILESYSTEMS_ROOT)
-    os_type, dirs = find_dirs(fs_root)
-    if os_type in [OperatingSystem.LINUX, OperatingSystem.OSX]:
-        return os_type, [dirs[0]]
+
+    # Linux, Unix and OS-X
+    file_root = fs_root.joinpath("file")
+    if file_root.exist():
+        os_type, dirs = find_dirs(file_root)
+        if os_type in [OperatingSystem.LINUX, OperatingSystem.UNIX ,OperatingSystem.OSX]:
```

Comment:
`find_dirs` can currently only return `OperatingSystem.LINUX`, `OperatingSystem.OSX` or `OperatingSystem.WINDOWS`. There shouldn't be a need to check against `OperatingSystem.UNIX`.